go.mod

```diff
@@ -205,7 +205,7 @@ replace (
 	github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
 
 	// use replace to force update grocksdb dependency in tm-db
-	github.com/linxGnu/grocksdb => github.com/linxGnu/grocksdb v1.7.15-0.20230222024938-b61261a9193b
+	github.com/linxGnu/grocksdb => github.com/yihuang/grocksdb v1.7.15-0.20230303073550-e611b544c0cf
```

Comment:
do we still replace? since already merged

Reply:
changed to upstream, still use replace to force indirect dependencies.